.gitlab/source_test/linux.yml

```diff
@@ -78,7 +78,7 @@ tests_deb-x64-py3:
     PYTHON_RUNTIMES: '3'
     CONDA_ENV: ddpy3
 
-tests_deb-x64-py3-fast:
+tests_deb-x64-py3-fast-v2:
```

Comment:
what's the point of this change?

Reply:
The point is I want to draw graph with the execution time of the tests using this heuristic. But if I do not rename the job name my statistics will get polluted by tests from not rebased PRs that use the old heuristic

Reply:
Could you add a tag instead ? Maybe add `--tags` to `EXTRA_OPTS`